### ISSUE TITLE
Weight decay in EMA?

### ISSUE THREAD

**Issue by bkj** [kekmodel/FixMatch-pytorch#15]
Hi --

Are you able to explain the motivation behind this line:
https://github.com/kekmodel/FixMatch-pytorch/blob/master/train.py#L494

Eg, when you use EMA, you're also shrinking the weights of the base model?

Thanks!

**Comment by LeeDoYup** [comment]
I think the weight decay is not conducted in optimizer, but in EMA update.

**Comment by kekmodel** [comment]
> I think the weight decay is not conducted in optimizer, but in EMA update.

That's right, this method is more like TensorFlow's method.
